docs/plugin_developers_guide/01_Fledge_plugins.rst

```diff
@@ -41,11 +41,109 @@ Plugins
++------------------------+------------------------+----------------------+-------------------------------------------------------+
+| enable                 | Filter |br|            | Enable/disable       | description: "<Describe plugin being enabled>" |br|   |
+|                        | Notification |br|      | plugin function      | type: "boolean"                                |br|   |
+|                        | Notification delivery  |                      | default: "false"                               |br|   |
+|                        |                        |                      | displayName: "<name for UI display>"           |br|   |
+|                        |                        |                      | order: "<order of display in UI>"                     |
++------------------------+------------------------+----------------------+-------------------------------------------------------+
+| asset                  | South |br|             | Asset where data     | description: "<name of asset being monitored>" |br|   |
+|                        | Notification           | fields are deposited | type: "string"                                 |br|   |
+|                        |                        |                      | default: ""                                    |br|   |
+|                        |                        |                      | displayName: "<name for UI display>"           |br|   |
+|                        |                        |                      | mandatory: "true"                              |br|   |
+|                        |                        |                      | order: "<order of display in UI>"                     |
++------------------------+------------------------+----------------------+-------------------------------------------------------+
+| source                 | North                  | Choice of data to    | description: "<Resource being forwarded>" |br|        |
+|                        |                        | forward from Foglamp | type: "enumeration"                  |br|             |
+|                        |                        | (readings or         | options: ["readings", "statistics"]  |br|             |
+|                        |                        | statistics)          | default: "readings"                  |br|             |
+|                        |                        |                      | displayName: "<name for UI display>" |br|             |
+|                        |                        |                      | order: "<order of display in UI>"                     |
++------------------------+------------------------+----------------------+-------------------------------------------------------+
+
+Fledge plugin methods
+---------------------
+Different plugin types (eg., north/south/...) have common and distinct APIs they must export.
```

Comment:
There are a common set for all plugins and then there are specifics for each type of plugin, I would suggest we are better documenting the specifics with the documentation on those plugin type rather than lumping them into one big table.